.gitignore

```diff
@@ -4,14 +4,7 @@ node_modules
 project/project
 project/metals.sbt
 target
-*.idea
```

Comment:
Isn't this files purpose to prevent these kind of files to slip into the commits?

Reply:
These should be in global .gitignore (~/.gitignore) instead of project's .gitignore.